packages/sprite-tiling/src/TilingSprite.ts

```diff
@@ -280,7 +283,7 @@ export class TilingSprite extends Sprite
      * @param {number} height - the height of the tiling sprite
      * @return {PIXI.TilingSprite} A new TilingSprite using a texture from the texture cache matching the frameId
      */
-    static fromFrame(frameId, width, height)
+    static fromFrame(frameId: string, width: number, height: number): TilingSprite
```

Comment:
@bigtimebuddy  - fromFrame and fromImage ... we deprecated these for Sprites and Textures in favour of just 'from' for v5, but looks like this got missed. Is there a case for doing that here too? Or wait until post TS conversion?

Reply:
I would rather do that as a follow up PR, but you’re right that was missed.